netlogo-gui/src/test/lab/gui/ProtocolEditableTests.scala

```diff
@@ -31,17 +31,4 @@ class ProtocolEditableTests extends AnyFunSuite {
       assert(protocol == editedProtocol)
     }
   }
-
-  test("protocol error on overflow") {
```

Comment:
After I changed some other files, this test started failing, but upon review I realized it was actually not written correctly, so for now I just removed it to make the tests pass. There are tests that it could be replaced with in the future, but they wouldn't be really equivalent.